src/pages/ScoreReport.vue

```diff
@@ -932,6 +942,16 @@ const shouldBeOutlined = (taskId) => {
   else return true;
 };
 
+const optionalAssessments = computed(() => {
+  return administrationInfo.value?.assessments.filter((assessment) => assessment.optional) || [];
+});
+
+const isOptional = (_taskId) => {
+  return optionalAssessments.value.some((assessment) => assessment.taskId === _taskId);
+};
```

Comment:
As with the other PRs, I think we need to pull the `optional` boolean from each assignment, rather than from the administration because it can be different from person to person.